compiler/compiler.go

```diff
@@ -2473,6 +2473,8 @@ func (c *Compiler) parseConvert(typeFrom, typeTo types.Type, value llvm.Value, p
 		switch elemType.Kind() {
 		case types.Byte:
 			return c.createRuntimeCall("stringToBytes", []llvm.Value{value}, ""), nil
+		case types.Rune:
+			return c.createRuntimeCall("stringToRunes", []llvm.Value{value}, ""), nil
 		default:
 			return llvm.Value{}, c.makeError(pos, "todo: convert from string: "+elemType.String())
```

Comment:
Both cases allowed by the Go spec are now supported, so it's a bug in the compiler if we get here. You can replace the `todo:` error with a panic. (Note: getting here would be a bug because when we get to SSA level the code has long been type checked and has already been verified as being valid Go).

Reply:
I was really not sure, what message to put there, please help me with that.

Reply:
What about `unexpected type in string to slice conversion`? It's not hugely important, but it's useful if it is somewhat descriptive.

Reply:
Sounds good, I've made the suggested changes, can you take a look?